CONTRIBUTING.md

```diff
@@ -14,3 +14,223 @@ If you are sending a pull request and it's a non-trivial change beyond fixing ty
 [the ICLA (individual contributor license agreement)](https://feedback.line.me/enquete/public/799-Z6ygVz3U). Please
 contact us if you need the CCLA (corporate contributor license agreement).
 
+### How to build
```

Comment:
Well I think "How to build" section isn't necessary to be added in `CONTRIBUTING.md` as we can expect people who contributes us reads README in prior to `CONTRIBUTING.md`.

Besides, "How to build" is a worth information not only for people who thinks to contribute but also for people who just wanna build this project manually, so I feel its more natural README to have it. 

However I'm not strong on this. As we're using a very popular build tool with very standard usage, "how to build" is very clear for most people, so either choice is fine for me :)